hack/boilerplate/boilerplate.py

```diff
@@ -22,142 +22,156 @@
 import re
 import sys
 
-
-SKIPPED_DIRS = ["Godeps", "third_party", ".git", "vendor", "examples", "testdata"]
+SKIPPED_DIRS = [
```

Comment:
Looks like this is irrelevant to this Change. can we move this out in a different PR

Reply:
The file was autoformated by `google/codefmt-vim` (which I think is uses `Autopep8` for python) when I updated the list of valid years. I can revert this file, disable the formatter, and redo the change to years if you prefer

Reply:
@tejal29 I reverted this and submitted the change as #949 instead